.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: kemenaran/rgbds:0.8.0
+      - image: ghcr.io/gbdev/rgbds:master
```

Comment:
Yay for using the official rgbds image! I have no will of maintaining one myself, so I'll even soon remove the whole `.circleci/images/rgbds/Dockerfile` file.

However, I'd be in favor of having an explicit version number here. Any reason not to have `rgbds:0.9.2`?

Reply:
Ok, I understand ^^

Let's go then. Will you be able to make a small PR moving from `master` → `0.9.3` when the fixed docker image will be up, or should I set a reminder to do it at some point?